vhdbuilder/packer/components.json

```diff
@@ -227,18 +227,15 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:*",
       "amd64OnlyVersions": [
-        "v0.6.0",
         "v1.0.2",
         "v1.1.1"
       ],
       "multiArchVersions": [
-        "v0.7.16",
```

Comment:
Determined these versions from _images.tpl file in RP. Please confirm if it's ok to remove these from the VHD.